src/fidesops/schemas/drp_privacy_request.py

```diff
@@ -22,7 +24,7 @@ class DrpPrivacyRequestCreate(BaseSchema):
 
     meta: DrpMeta
     regime: Optional[DrpRegime]
-    exercise: DrpAction
+    exercise: List[DrpAction]
```

Comment:
DRP spec dictates that this is a list. So we'll take in a list even though we don't support > 1 item